### PR TITLE
TDR-3690 Add File UUID to the filemetadata table

### PR DESCRIPTION
**Pull request by thanhz**
To be included in the bagit export

### CODE REVIEW

**Comment by TomJKing** [comment]
@Tom-Hallett / @thanhz Tom H would you mind reviewing this change by Thanh? Good chance to use you knowledge so far of the API.

File: src/test/resources/application.conf
Comment (Tom-Hallett) [inline src/test/resources/application.conf:1]: Suggest changing to a different alternate http port (e.g. 8008) for consistency

File: src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala [hunk 1]
```diff
@@ -216,8 +216,8 @@ class FileRouteSpec extends TestContainerUtils with Matchers with TestRequest {
   }
 
   private def getReferencesMockServer(additionalRefs: Int = 0): WireMockServer = {
-    val wiremockServer = new WireMockServer(8080)
-    WireMock.configureFor("localhost", 8080)
+    val wiremockServer = new WireMockServer(2234)
```
Comment (Tom-Hallett) [inline src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala:219]: As above - suggest changing to a different alternate http port (e.g. 8008) for consistency

**Comment by Tom-Hallett** [comment]
Looks good. One very minor suggestion.

File: src/test/resources/application.conf
Comment (thanhz) [reply src/test/resources/application.conf:1]: As suggested. I've changed the port to `8008`

**Pull request merged**